pkg/controllers/common/constants.go

```diff
@@ -116,6 +116,14 @@ const (
 	TemplateGeneratorMergePatchAnnotation = FederateControllerPrefix + "template-generator-merge-patch"
 
 	LatestReplicasetDigestsAnnotation = DefaultPrefix + "latest-replicaset-digests"
+
+	HPAScaleTargetRefPath = DefaultPrefix + "scale-target-ref-path"
+)
+
+// The following consts are labels key-values used by Kubeadmiral controllers.
+
+const (
+	CentralizedHPAEnableKey = DefaultPrefix + "centralized-hpa-enabled"
```

Comment:
Why a separate block with duplicated comment?

Reply:
It can be merged into the previous block.

Reply:
The code has been updated.